python/fedml/api/modules/storage.py

```diff
@@ -287,7 +303,7 @@ def _upload_part(url,part_data,session):
     return response
 
 
-def _upload_chunk(presigned_url, chunk, part, pbar=None, max_retries=20,session=None):
+def _upload_chunk(presigned_url, chunk, part, pbar=None, max_retries=20,session=None, byte_data_flag= False):
```

Comment:
We aren't using the flag in this function. Why do we pass it here?

Reply:
Removed